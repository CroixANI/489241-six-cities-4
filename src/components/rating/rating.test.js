import React from 'react';
import renderer from 'react-test-renderer';

import Rating from './rating';

it(`Render Rating component`, () => {
  const tree = renderer
    .create(<Rating rating={4.25} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
