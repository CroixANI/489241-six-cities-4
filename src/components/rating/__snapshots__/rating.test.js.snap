// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Rating component 1`] = `
<div
  className="place-card__stars rating__stars"
>
  <span
    style={
      Object {
        "width": "80%",
      }
    }
  />
  <span
    className="visually-hidden"
  >
    Rating
  </span>
</div>
`;
